src/app/script/app_theme_object.cpp

```diff
@@ -54,39 +104,15 @@ int ThemeColor_index(lua_State* L)
 
 int Theme_styleMetrics(lua_State* L)
 {
-  [[maybe_unused]]
```

Comment:
Don't remove `[[maybe_unused]]` (the same for all the following cases).

Reply:
This is the only attribute I removed, so I'm not sure what do you mean with "all the following cases". Anyway thank you for clarifying in another comment why this attribute is needed.